dist/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To Do List</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <main>
+      <header>
+        <div class="header">
+          <div class="todo">
+            <h1>Today's To Do</h1>
+            <div class="image"></div>
+          </div>
+        </div>
+        <hr />
+        <form action="#">
+          <input type="text" placeholder="Add to your list..." class="add" />
+          <div class="enter"></div>
+        </form>
+        <hr />
+        <ul class="list"></ul>
+      </header>
+
+      <div class="footer">
+        <h3>Clear all completed</h3>
+      </div>
+    </main>
+  </body>
+</html>
```

Comment:
- [ ] [OPTIONAL] I recommend improving your design to make your project stand out 😉. You can get inspiration from [this website](https://dribbble.com/) 🍀